wordpress-mobile/WordPress-Android
# Add news card to insights

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/9687

This PR adds the News card to Insights management. The card shows a button to dismiss (which should hide it) and a button to "Try it now" which scroll down to the "Edit" button. Once you click on the "Edit" button, we assume you know you tried Insights management and we hide the News card. The designs are different than in the mentioned issue but you can check zeplin to see they are correct. 

I tried to split the issue more and I created this PR - https://github.com/wordpress-mobile/WordPress-Android/pull/9893 - but the rest seems pretty atomic. 

To test:
* Log in to the app
* Go to Stats/Insights
* You see a "News" card on top of the list
* Click on "Try it now"
* The app scrolls down to the "Edit" button
* Click on the "Edit" button and go back
* The "News" card is hidden

To test 2:
* Log in to the app
* Go to Stats/Insights
* You see a "News" card on top of the list
* Click on "Dismiss"
* The "News" card is hidden

![Screenshot_1558428650](https://user-images.githubusercontent.com/1079756/58081977-5a413d00-7bb6-11e9-8e49-b0af8cba6875.png)

Update release notes:
